src/main/java/com/zebrunner/carina/api/apitools/validation/XmlComparator.java

```diff
@@ -71,7 +70,9 @@ public static void nonStrictOrderCompare(String actualXmlData, String expectedXm
                     }
                     return outcome;
                 }).checkForSimilar().build();
-        Assert.assertFalse(differences.hasDifferences());
+        if(differences.hasDifferences()) {
+            throw new RuntimeException("Has difference.");
```

Comment:
how about adding details about actual and expected data? if it is pretty huge xml we can raise exception like:
```
nonStrictOrderCompare has difference. enabled debug log level for details.
```

and before raise log expected and actual string on debug layer.